### DISCUSSION TITLE
Change the default database GUI

### DISCUSSION THREAD

**Discussion by codeallday31** [leokhoa/laragon#767] [Q&A] [answered]
can someone help me how to change the database GUI, as I remember there's a way to change the default database GUI by changing the laragon.ini file. 

thanks in advance 😄

**Comment by popkilli** [comment, marked as answer]
add the line below into laragon.ini and restart Laragon, for my case example i'm using SQLYog.

[ui]
Database=SQLyog: D:\Laragon\bin\sqlyog\SQLyog.exe

change the directory path accordingly,

![Snipaste_2024-05-08_23-18-28](https://github.com/leokhoa/laragon/assets/94550728/a87ff6b0-f040-4991-9712-fb5a75e2aa45)

**Reply by codeallday31** [reply]
Thank you brother
